locations/spiders/bexley.py

```
@@ -0,0 +1,40 @@
+import re
+import json
+
+
+
+class BexleySpider(JSONBlobSpider):
+    name = "bexley"
+    item_attributes = {
+        "brand": "Bexley",
+        "brand_wikidata": "Q101247434",
+    }
+    #the start page must be any shop url
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)